File: src/main/java/net/softbell/bsh/iot/dto/bshp/v1/ItemInfoV1DTO.java
package net.softbell.bsh.iot.dto.bshp.v1;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : BSHPv1 전용 Items Info DTO
 */
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class ItemInfoV1DTO
{
	private Byte controlMode;
	private Byte pinId;
	private Integer pinMode;
	private Integer pinType;
	private String pinName;
}

File: src/main/kotlin/net/softbell/bsh/iot/dto/bshp/v1/ItemInfoV1Dto.kt
package net.softbell.bsh.iot.dto.bshp.v1

/**
 * @author : Bell(bell@softbell.net)
 * @description : BSHPv1 전용 Items Info DTO
 */
data class ItemInfoV1Dto (
    var controlMode: Byte,
    var itemIndex: Byte,
    var itemMode: Int,
    var itemCategory: Int,
    var itemType: Int,
    var itemName: String
)